docs/en/aws-elastic-serverless-forwarder-configuration.asciidoc

```diff
@@ -288,10 +290,17 @@ inputs:
 
 You can define `inputs.[].expand_event_list_from_field` as a string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
 
+If you want to inject in the expanded events, fields from the root of the JSON object that you are expanding from, you can define `inputs.[].root_fields_to_add_to_expanded_event`. The config takes two different type of value:
+
+. the literal string "**all**", in order to inject all the fields at root level (but for the one you are expanding events from): `root_fields_to_add_to_expanded_event: "all"`
```

Comment:
I think removing some unnecessary words here makes the intention a little clearer.
```suggestion
. the literal string "**all**" to inject all root fields (except the field you are expanding events from). For example, `root_fields_to_add_to_expanded_event: "all"`
```